2200100/문제5.py

```diff
@@ -0,0 +1,19 @@
+T = int(input())
+cnt = 0
+cnt1 = 0
+for t in range(T):
+    num = list(map(int,input().split()))
+    for n in num:
+        cnt1 += 1
+        if cnt1%2 == 1 :
+            cnt += n*2
+        elif cnt1%2 == 0 :
+            cnt += n
+    if cnt%10 != 0 :
+        b = 10 - (cnt%10)
+    elif cnt%10 == 0:
+        b = 0
```

Comment:
깔끔한 풀이네요! 👍 
```suggestion
    if cnt%10 != 0 :
        b = 10 - (cnt%10)
    else:
        b = 0
```
10으로 나눈 나머지가 0인 경우와 0이 아닌 경우 2가지 밖에 없으므로 `else` 를 활용하셔도 좋을 것 같습니다 😄